src/site-switcher/site-switcher.html

```diff
@@ -0,0 +1,13 @@
+<div class="dropdown miq-siteswitcher">
+  <button class="btn btn-link dropdown-toggle" type="button" id="domain-switcher" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+    <span class="fa fa-lg fa-th miq-siteswitcher-icon"></span>
+  </button>
+  <div class="dropdown-menu dropdown-menu-right" aria-labelledby="domain-switcher">
+    <div ng-repeat="site in ctrl.sites" class="miq-siteswitcher-entry">
+      <a class="miq-siteswitcher-link" href="{{site.url}}" target="_blank">
+        <span title="{{site.tooltip}}" class="fa fa-3x {{site.iconClass}}"></span>
```

Comment:
Please use `ng-class` instead of `class="{{iconClass}}", I know that it's not this case now, but we might send in future and object or array and it would be broken.
```html
...
<span title="{{site.tooltip}}" class="fa fa-3x" ng-class="site.iconClass"></span>
...
```